x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java

```diff
@@ -279,6 +279,7 @@ public final void test() throws Throwable {
                 "CSV tests cannot correctly handle the field caps change",
                 testCase.requiredCapabilities.contains(EsqlCapabilities.Cap.SEMANTIC_TEXT_FIELD_CAPS.capabilityName())
             );
+            assumeFalse("fork disabled for csv tests", testCase.requiredCapabilities.contains(EsqlCapabilities.Cap.FORK.capabilityName()));
```

Comment:
I *think* CsvTests should be able to run these. I did a bunch with InlineStats. At least, I think I did. It's been a while.

Reply:
ah yes, that's better. done. [38c0577](https://github.com/elastic/elasticsearch/pull/121948/commits/38c05778e2296ce01240f1adcc1c84d589f3576e)